test/hare_test

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+import argparse
```

Comment:
Why do we need this script?

Reply:
This script will get called as follows,
`/opt/seagate/cortx/hare/libexec/hare_test execute --plan sanity --cdf_path /var/lib/hare/cluster.yaml`

Our 'hare_setup test' will call this command in above format.